.github/workflows/staging.yml

```diff
@@ -0,0 +1,51 @@
+name: Build and Deploy Site
+
+on:
+  push:
+    branches:
+      - staging
+
+jobs:
+  build-and-deploy-site:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+          fetch-depth: 0
+
+      - name: Setup Hugo
+        uses: peaceiris/actions-hugo@v2
+        with:
+          hugo-version: 'latest'
+          extended: true
+
+      - name: Build site with Hugo
+        run: hugo --minify
+
+      - name: Check HTML
+        uses: chabad360/htmlproofer@master
+        with:
+          directory: "./public"
+          arguments: --only-4xx --check-favicon --check-html --assume-extension --empty-alt-ignore --disable-external
+        continue-on-error: true
+
+      - name: Add a robots.txt to disallow crawling staging
+        run: |
+          mkdir ~/.buildout
+          cat > ./public/robots.txt <<EOF
+          User-agent: *
+          Disallow: /
+
+          EOF
+      - name: Deploy to swingconnction staging
```

Comment:
```suggestion
      - name: Deploy to swingconnection staging
```
This is the way to actually suggest the change 

![image](https://user-images.githubusercontent.com/17002572/212773357-25bab053-7a61-4c94-aba0-2e1ae5a57108.png)

![image](https://user-images.githubusercontent.com/17002572/212773371-c43b3aa6-12ff-4daa-a37a-b9e891003874.png)